Dockerfile

```diff
@@ -1,11 +1,22 @@
 FROM php:7.1-apache-stretch
 
+ENV LD_LIBRARY_PATH /usr/local/instantclient
+
+# Setup the required extensions.
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install NVM and the current (as of 26/02/2019) LTS version of Node.
+ENV NVM_DIR /usr/local/nvm
+ENV NODE_VERSION  lts/carbon
+RUN mkdir -p $NVM_DIR && \
```

Comment:
Note: `/usr/local/nvm` means here that _build_ should be performed using a privileged user, which is **not** preventing _running_ the resulting image as low privileged user (https://github.com/moodlehq/moodle-php-apache/issues/1#issuecomment-467254000)

Reply:
Yup: It only means that any calls to `nvm install` must be run privileged, but not that actual use of `node`. Setting a different version in the `NODE_VERSION` environment variable prior to starting the container will install.

That said, I've made a change to add nvm.sh to the /etc/bash.bashrc so that all users can use it, removed the REPL log, and made sure that it can be used by the www-data user.
I've also added an NVM wrapper to /usr/local/bin/nvm which will source the nvm.sh file and run nvm for you so that you can run it using `docker run`.

Reply:
👍